src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(...args) {
```

Comment:
The `adder` function is intended to accept a single argument at a time according to the task description. Using the rest parameter (`...args`) here allows it to accept multiple arguments, which deviates from the specified behavior. Consider modifying the function to accept only one argument at a time.